docs/docs/overview/security.md

```diff
@@ -25,23 +23,21 @@ Our container is also based on [Alpine Linux](https://alpinelinux.org/) which ha
 One of the biggest security risks for an application is keeping credentials around longer than they are needed. Because Secretless Broker off-loads credential management from the app itself, there will be a window of time where such credentials are retrieved and injected into the backend connection but majority of the time, your credentials would be only stored in your providers that are built for their long-term storage. Since each listener/handler combination is responsible for credential lifecycles these are the lifecycles of credentials for each of the built-in listeners:
```

Comment:
this text

> there will be a window of time where such credentials are retrieved and injected into the backend connection but majority of the time, your credentials would be only stored in your providers that are built for their long-term storage

needs to be edited.

I would suggest "Secretless Broker centralizes the credential management functionality so that the application no longer has to worry about it, and in doing so is able to ensure that credentials are not stored in memory for longer than they are needed to open a new connection."

Reply:
We should also be consistent here as we are in the rest of the documentation with capitalizing the key terms - Listener, Handler, Credential Provider.